examples/cifar10_cnn.py

```diff
@@ -15,10 +15,20 @@
 from keras.layers import Dense, Dropout, Activation, Flatten
 from keras.layers import Conv2D, MaxPooling2D
 
+import os
+import pickle
+from IPython.utils.path import ensure_dir_exists
```

Comment:
Thank you for the code review. I was using this dependency to simplify the call to check dir and make if it doesn't. I can switch to os and add the check in the example.